src/NzbDrone.Core/Parser/LanguageParser.cs

```diff
@@ -146,6 +148,23 @@ public static Language ParseSubtitleLanguage(string fileName)
 
             return Language.Unknown;
         }
+        
+        public static IEnumerable<string> ParseLanguageTags(string fileName)
+        {
+            try
+            {
+                var simpleFilename = Path.GetFileNameWithoutExtension(fileName);
+                var tagMatches = SubtitleLanguageRegex.Matches(simpleFilename);
+                var languageTags = from Match tagMatch in tagMatches select tagMatch.Groups["tags"].Value.ToLower();
```

Comment:
I got some weird issue here as the code stopped returning multiple matches. The `.Matches()` just started behaving exactly like `.Match`, only giving the last match. Yesterday, it worked just fine.

I am not aware of any changes which could change the behavior, tried to clean and rebuild Sonarr, but it's still happening. Do you have any ideas?